uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -18,7 +18,7 @@ define([
     '$filter', '$uibModal', '$dialog', '$timeout', 'AssignmentService', 'DocumentService', 'ContactService', 'FileService', 'data',
```

Comment:
Name of the file should be modal-document.controller.js. Please amend this file according to the styleguide as much as possible, including utilisation of $inject, **if possible** getting rid of $scope, private functions + vm.function, order of things etc.